.github/workflows/deploy.yml

```diff
@@ -58,7 +58,7 @@ jobs:
     outputs:
       deployment_url: ${{ steps.deployment-url.outputs.url }}
     steps:
-      - uses: actions/checkout@v3
+      - uses: actions/checkout@v3.5.2
```

Comment:
I’m curious. Is there any value in pinning these GitHub provided actions?

Reply:
No strong opinion. I’ve often just been using major versions for GitHub provided actions, but likely not with a very good reason.